src/Components/PastMenu.js

```diff
@@ -0,0 +1,254 @@
+//React components
+import React from "react";
+import Carousel from "react-bootstrap/Carousel";
+
+//indochine images
+import chickpeaTofuSalad from "../images/indochine/starters/chickpea-tofu-salad.jpg";
+import gallopingHorses from "../images/indochine/starters/galloping-horses.jpg";
+import grilledNumAnsom from "../images/indochine/starters/grilled-num-ansom.jpg";
+import miniBanhMi from "../images/indochine/starters/mini-banh-mi.jpg";
+
+import cauliflowerLaab from "../images/indochine/mains/cauliflower-laab.jpg";
+import greenCurryOfFish from "../images/indochine/mains/green-curry-of-fish.jpg";
+import PanaengDuckCurry from "../images/indochine/mains/panaeng-duck-curry.jpg";
+import saiOuaPorkSausage from "../images/indochine/mains/sai-oua-pork-sausages.jpg";
+import shrimpPasteRelish from "../images/indochine/mains/shrimp-paste-relish.jpg";
+import lemongrassJasmineRice from "../images/indochine/mains/lemongrass-jasmine-rice.jpg";
+
+import smokedIceCreamWithCakeCroutons from "../images/indochine/smoked-ice-cream-with-cake-croutons.jpg";
+
+//nusantara images
+import bandungSiomay from "../images/nusantara/starters/bandung-siomay.jpg";
+import nasibal from "../images/nusantara/starters/nasibal.jpg";
+import pepesTahuJamur from "../images/nusantara/starters/pepes-tahu-jamur.jpg";
+import sotoAyamIstimewa from "../images/nusantara/starters/soto-ayam-istimewa.jpg";
+
+import babiRawon from "../images/nusantara/mains/babi-rawon.jpg";
+import bebekBetutu from "../images/nusantara/mains/bebek-betutu.jpg";
+import ikanPesmol from "../images/nusantara/mains/ikan-pesmol.jpg";
+import nasiLiwet from "../images/nusantara/mains/nasi-liwet.jpg";
+import plecingSayur from "../images/nusantara/mains/plecing-sayur.jpg";
+import sambalHijau from "../images/nusantara/mains/sambal-hijau.jpg";
+import vegetableSodhi from "../images/nusantara/mains/vegetable-sodhi.png";
+
+import notKlepon from "../images/nusantara/not-klepon.jpg";
+
+class PastMenu extends React.Component {
+  pastMenuDict = {
```

Comment:
Same here. All static content, might as well define this outside this component or in an own file.

Reply:
I would envision some kind of function, where you can pass a date range as argument. The function would then dynamically import images and return this data structure for the menu of that date for you. This PastMenu component would make use of that